<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,6">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,2,0,0,6">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item row="4" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>My new game</string>
            </property>
            <property name="placeholderText">
             <string>Game name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item row="5" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Number of players</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Delay (sec)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Timeout (sec)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox">
            <property name="accessibleName">
             <string/>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="spinBox">
            <property name="maximum">
             <number>600</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="horizontalSlider_2">
            <property name="statusTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="accessibleName">
             <string/>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="maximum">
             <number>600</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSlider" name="horizontalSlider">
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Maps</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>label_2</zorder>
         <zorder>horizontalSlider_2</zorder>
         <zorder>label_4</zorder>
         <zorder>comboBox_2</zorder>
         <zorder>label</zorder>
         <zorder>horizontalSlider</zorder>
         <zorder>lineEdit</zorder>
         <zorder>comboBox</zorder>
         <zorder>doubleSpinBox</zorder>
         <zorder>spinBox</zorder>
         <zorder>label_3</zorder>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="GView" name="graphicsView"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Create Game</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0">
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="GView" name="graphicsView_2"/>
       </item>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>81</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Join to random game</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="8,1,1">
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>Leave</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Center on me</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <widget class="GView" name="graphicsView_3"/>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GView</class>
   <extends>QGraphicsView</extends>
   <header>GView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
